README.rst

```diff
@@ -0,0 +1,43 @@
+Mozzarella
+==========
+
+This website is a website for clubs and student organizations.
+
+Mozzarella It is written in Python and uses the powerful web framework
+`TurboGears`.
+
+.. TurboGears: http://turbogears.com/
+
+Getting Started
```

Comment:
Confusing header... is this where I start to set it up in production?

"Setting up your Development Environment" is better.